deployment/terraform/scripts/rds_shared_snapshot.sh

```
@@ -0,0 +1,32 @@
+  test -f $(which jq) || error_exit "jq command not detected in path, please install it"
+  test -f $(which aws) || error_exit "aws cli not detected in path, please install it"
+}
+
+function parse_input() {
+  eval "$(jq -r '@sh "export DB_INSTANCE_IDENTIFIER=\(.db_instance_identifier)"')"
+  if [[ -z "${DB_INSTANCE_IDENTIFIER}" ]]; then export DB_INSTANCE_IDENTIFIER=none; fi
+}
+
+function get_latest_snapshot_id() {
+  instance_identifier="${DB_INSTANCE_IDENTIFIER}"
+  export SNAPSHOT_ID=$(aws rds describe-db-snapshots --include-shared | jq --arg instance_identifier "$instance_identifier" '[.DBSnapshots[] | select(.DBInstanceIdentifier == $instance_identifier)][-1]' | jq '.DBSnapshotArn' |  tr -d '"\\')
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)